package.json

```diff
@@ -25,6 +25,8 @@
 		"lint": "lerna run lint",
 		"lint:fix": "lerna run lint:fix",
 		"test": "lerna run test",
+		"test:node": "lerna run test:node",
+		"test:browser": "lerna run --concurrency 1 test:browser",
```

Comment:
if lerna runs 2 commands at the same time, http-servers were crashing since they were set to use the same port. Setting concurrency to 1 ensures that we run one test command at a time.

Reply:
If we find we need to speed things up we can always run the servers on different ports at a later point.

Reply:
Since we set the ports to different one for each test, this can be higher?

Reply:
@shuse2 good catch. Thanks. I will remove the flag, so lerna can execute tests by its own algorithm.